stacks/nodejs-nuxtjs/devfile.yaml

```diff
@@ -1,17 +1,15 @@
 schemaVersion: 2.1.0
 metadata:
-  description:
-    'Nuxt is the backbone of your Vue.js project, giving structure to build your project with confidence while keeping flexibility.
-    Its goal is to help Vue developers take advantage of top-notch technologies, fast, easy and in an organized way.'
+  name: nodejs-nuxtjs
   displayName: Nuxt.js
+  description: Node.js 16 application with Nuxt.js
```

Comment:
I like the original description for the stack description. Maybe just add the version number in